mldatafind/segments.py

```diff
@@ -1,14 +1,28 @@
 from typing import Iterable
 
-from gwpy.segments import DataQualityDict, SegmentList
+from gwpy.segments import DataQualityDict, DataQualityFlag, SegmentList
 
 from mldatafind.authenticate import authenticate
 
+# TODO: Not exactly sure what these flags
+OPEN_DATA_FLAGS = ["H1_DATA", "L1_DATA", "V1_DATA"]
```

Comment:
Would it be worthwhile to allow users to pass just the IFO name the same way they do `fetch_timeseries` and `find`, and we can map to the flag name under the hood? Might save people from having to memorize a bunch of nomenclature, but then again if practitioners who are used to this will find it confusing then maybe its not worthwhile

Reply:
Yeah I see your point. I think keeping consistency with what is on gwosc makes the most sense. I think the reason they include the `DATA` is because there are other segments that are available to query via the same api ( which I think are actually the vetoes). 

 For strain there is no ambiguity so they use the IFO names, which I agree is much cleaner.

Reply:
Ok makes sense, let's leave as-is then